index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="C:\Users\Genaro\Desktop\css\bootstrap.css">
+<link rel="stylesheet" href="C:\Users\Genaro\Desktop\css\bootstrap-grid.css">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>GirlDevelopIt</title>
+</head>
+
+<body>
+    <br>

```

Comment:
Use a bootstrap `container` instead of a break, which it's not a good practice. Read more: https://getbootstrap.com/docs/4.0/layout/overview/#containers